vmdb/app/models/ems_kubernetes.rb

```diff
@@ -20,8 +20,16 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
-  def self.raw_connect(hostname, port, _service = nil)
-    kubernetes_connect(hostname, port)
+  def supported_auth_types
+    %w(basic)
+  end
+
+  def supports_authentication?(authtype)
```

Comment:
please s/authtype/auth_type - it's more readable and it's more ruby kind of style.